cypress/e2e/login.cy.js

```diff
@@ -0,0 +1,82 @@
+/**
+ * - Login spec
+ *   - should display login page correctly
+ *   - should display alert when email is empty
+ *   - should display alert when password is empty
+ *   - should display alert when email and password are wrong
+ *   - should display homepage when email and password are correct
+ */
+
+describe('Login spec', () => {
+  beforeEach(() => {
+    cy.visit('http://localhost:3000/login');
+  });

```

Comment:
```suggestion
  beforeEach(() => cy.visit('http://localhost:3000/login') );

```
Akan lebih mudah dibaca dan ringkas jika ditulis seperti ini.